manifests/ols-redis.yaml

```diff
@@ -46,7 +46,7 @@ spec:
     spec:
       containers:
       - name: redis
-        image: ${OLS_REDIS_IMAGE}
+        image: docker.io/redis/redis-stack:latest
```

Comment:
I am not sure if its safe to keep docker here. I often run into `download limit exceeded` error while pulling images from a docker account.

Reply:
As an alternative, I can only suggest cloning Redis image somewhere to quay.io

Reply:
can we use https://quay.io/repository/sclorg/redis-6-c9s ?

that's the redhat centos streams distribution of redis.

Reply:
This redhat centos image has just `redis` but not other things like insights etc that we have redis-stack image. Can't we have a dedicated repo for OLS project that we can to store our images?

Reply:
> Can't we have a dedicated repo for OLS project that we can to store our images in quay.io?

yes, we can, but then we need to setup something to push images to it, like we did for the OLS image.  Whether that is just mirroring an image, or it's us having our own Containerfile that extends from the base image we want and possibly does other configuration of the image.  The latter is probably the best approach because ultimately when we productize this, we're going to need our own Containerfile that lets us build the image.

So the easiest thing to do is introduce a Containerfile.redis to this repo.  Please create a "images/redis" folder structure to contain it.  Once we've landed the containerfile we can update openshift/release to build+publish the image to quay.io like we are doing with OLS.

(and then we're going to want to move the existing Containerfile under that images folder structure too, but that'll take a little more coordination to update openshift/release)

Reply:
> So the easiest thing to do is introduce a Containerfile.redis to this repo. Please create a "images/redis" folder structure to contain it. Once we've landed the containerfile we can update openshift/release to build+publish the image to quay.io like we are doing with OLS.

According to this plan, @vbelouso  can you create a "images/redis" folder and write up a Containerfile to publish our own redis image to quay.io to start with? 
```
docker.io/redis/redis-stack:latest
EXPOSE 6379
EXPOSE 8001
```

Reply:
I didn't intend to make that a prereq to this PR landing.  We can live w/ the docker.io location temporarily.